test/system/src/main/java/io/pravega/test/system/framework/Utils.java

```diff
@@ -45,8 +48,17 @@ public static String getConfig(final String key, final String defaultValue) {
     }
 
     public static Service createZookeeperService() {
-        return DOCKER_BASED ? new ZookeeperDockerService("zookeeper")
-                : new ZookeeperService("zookeeper");
+        TestExecutorFactory.TestExecutorType r = TestExecutorFactory.getTestExecutionType();
+        log.debug("Test executor type is {}", r);
+        switch (r) {
+            case DOCKER:
+                return new ZookeeperDockerService("zookeeper");
+            case K8:
```

Comment:
For consistency, kubernetes has 10 letters, and I see people replace the 8 letters in the middle with the digit 8, but the `K` and the `s` still need to be there, so it should be `K8s` or `k8s` if you don't like the capital `K`.

Reply:
used K8s.